Extensions/rnaoptions.hh

```diff
@@ -137,6 +138,8 @@ class Opts {
     size_t checkpoint_interval;  // default interval: 3600s (1h)
     boost::filesystem::path  checkpoint_out_path;  // default path: cwd
     boost::filesystem::path  checkpoint_in_path;  // default: empty
+	std::string user_file_prefix;
```

Comment:
is the extra indent intentional?

Reply:
> can you list all external datatypes that are affected, such that I can easily check? Maybe I can refactor such that only int indices are provided but not the actual sequence

`mfecovar_macrostate`, `answer_ali_pfunc_macrostate`, `answer_macrostate_mfe` and `answer_macrostate_pfunc` (defined in [typesRNAfolding.hh](https://github.com/jlab/fold-grammars/blob/master/Extensions/typesRNAfolding.hh))

Reply:
No it isn't. I will fix that.